### PR TITLE
Add topics to flatten-array

### LINKED ISSUE #898: flatten-array: add topics
As part of the update to the exercism website, we want to add topics to each exercise. This should be topics that will most likely be relevant to the exercise even though you might be able to solve it without encountering that topic. There is a [list of topics](https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt) used across all exercism tracks but you can add track specific ones as well if needed.

See issue #523 for a discussion about adding topics. There are also some exercises (e.g. hello-world and two-fer) that already have topics so you can look at them as examples. The topics need to be added to the entry for the relevant exercise in `config.json`.

Please comment on this issue or reference it in your pull request (using #issue-number) if you're going to be working on it. If you put "fixes #issue-number" in the pull request description then it will also automatically close this issue when the pull request gets merged :)

### PR DESCRIPTION
**Pull request by alexescg**
fixes #898 

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/xjava/blob/master/POLICIES.md#event-checklist)

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -350,7 +350,12 @@
       "core": true,
       "difficulty": 5,
       "slug": "flatten-array",
-      "topics": null,
+      "topics": [
+      	"arrays",
+	"lists",
+	"loops",
+	"recursion"
```
Comment (FridaTveit) [inline config.json]: Great suggestions! :) The indentation is a bit wonky though. All the topics should be indented two spaces in from `topics` :) Does that make sense?

Comment (alexescg) [reply config.json]: Yeah! I’ll take that into consideration for future PRs. And yeah I can fix identation, thats not a problem

**Comment by FridaTveit** [comment]
Will merge as soon as the build finishes :)

**Comment by alexescg** [comment]
Awesome! Thank you :D

**Pull request merged**